ui/src/app/pages/workspace/workspace-wrapper.tsx

```diff
@@ -133,156 +132,154 @@ const MultiRegionWorkspaceNotification = () => {
   );
 };
 
-export const WorkspaceWrapper = fp.flow(withCurrentWorkspace())(
-  ({ workspace, hideSpinner }) => {
```

Comment:
this workspace is wrong. It is the previous workspace from withCurrentWorkspace().